index.bs

```diff
@@ -5822,6 +5822,12 @@ Since certificates are sizable relative to the storage capabilities of typical a
 
 Note: In order to interoperate, user agents storing large blobs on authenticators using [[!FIDO-CTAP]] are expected to use the provisions detailed in that specification for storing [=large, per-credential blobs=].
 
+Note: [[!FIDO-CTAP]] authenticators only support this extension for [=discoverable credentials=],
+and might return an error unless
+<code>{{PublicKeyCredentialCreationOptions/authenticatorSelection}}.{{AuthenticatorSelectionCriteria/residentKey}}</code>
+is set to {{ResidentKeyRequirement/preferred}} or {{ResidentKeyRequirement/required}}.
+This is not a general requirement for all kinds of authenticators.
```

Comment:
(Responding to https://github.com/w3c/webauthn/pull/1622#discussion_r661625908 with wider suggestion scope)

Good point. :) Although we might not want to suggest that CTAP is the only _possible_ roaming authenticator protocol, so how about this?

```suggestion
Note: [=Roaming authenticators=] that use [[!FIDO-CTAP]] as their cross-platform transport protocol only support this [=largeblob/Large Blob=] extension for [=discoverable credentials=],
and might return an error unless
<code>{{PublicKeyCredentialCreationOptions/authenticatorSelection}}.{{AuthenticatorSelectionCriteria/residentKey}}</code>
is set to {{ResidentKeyRequirement/preferred}} or {{ResidentKeyRequirement/required}}.
However, [=authenticators=] that do not utilize [[!FIDO-CTAP]] do not necessarily restrict this extension to [=discoverable credentials=].
```

Reply:
Ok, LGTM, thx @emlun !